internal/storage/fake/testing/bucket_tests.go

```diff
@@ -4458,3 +4456,110 @@ func (t *cancellationTest) ReadObject() {
 				HasSubstr("request canceled"))))
 	ExpectLt(time.Since(before), 50*time.Millisecond)
 }
+
+func (t *listTest) TestListForUnsupportedNames() {
+	AssertEq(
```

Comment:
Blocks should be
```
create
inputs

Method call

assertions
```
should we remove unnecessary lines in between for readability perspective?

Reply:
obsolete comment. test removed.